src/libsync/logger.cpp

```diff
@@ -20,9 +20,12 @@
 #include <qmetaobject.h>
 
 #include "csync.h"
+#include "csync_log.h"
 
 namespace OCC {
 
+Q_LOGGING_CATEGORY(lcCsync, "sync.csync", QtInfoMsg)
```

Comment:
That one I kept since I'd like to be able to set `QT_LOGGING_RULES="sync.*=true"`, and I think csync plays a role at the same level as `sync.engine` and `sync.propagator`.

Doing `QT_LOGGING_RULES="*=true"` is a bit annoying because it also includes `qt.*`, so I opted for `gui` and `sync` as top-level rules and didn't parent everything under `oc.`.